src/MediaCT/ruleset.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0"?>
 <!--
 /**
- * Copyright MediaCT. All rights reserved.
- * https://www.mediact.nl
+ * Copyright Youwe. All rights reserved.
+ * https://www.youweagency.com
  */
 -->
 <ruleset name="Mediact">
```

Comment:
Shouldn't this name be updated? (and the description below)

Reply:
no not here, since this is the deprecated one.